src/assets/Generator.Shared/HttpPipelineExtensions.cs

```diff
@@ -96,7 +96,7 @@ public static Response<bool> ProcessHeadAsBoolMessage(this HttpPipeline pipeline
             }
         }
 
-        private static (CancellationToken CancellationToken, ErrorOptions ErrorOptions) ApplyRequestContext(RequestContext? requestContext, HttpMessage message)
+        private static (CancellationToken CancellationToken, ErrorOptions ErrorOptions) ApplyRequestContext(RequestContext? requestContext)
```

Comment:
I think we will use message again in the future - would it be possible to suppress the error we're getting for WebPubSub in the code temporarily?

Reply:
Since we are not using it and it's a private method, can we not just remove the parameter and bring it back when needed?